integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
@chadmiller Well like I said, I figured that out myself but I forgot to edit the code. It was a simple mistake that I'm glad was caught, and I'm thankful for that. That's all.

Now, what do I do now with this directory and this snapcraft.yaml file? See comment #4 on this line comment string(if that makes any sense).